README.zh-CN.md

```diff
@@ -12,38 +12,103 @@
 </div>
 
 ----
-
-<a href="https://ava.antv.antgroup.com"><img src="https://mdn.alipayobjects.com/huamei_qa8qxu/afts/img/A*rXVYRJ0EMDsAAAAAAAAAAAAADmJ7AQ/original" align="left"  width="180"  hspace="10" vspace="6"></a>
-
-**AVA** (<img src="https://mdn.alipayobjects.com/huamei_qa8qxu/afts/img/A*QzIsSrfsCW0AAAAAAAAAAAAADmJ7AQ/original" width="16"> Visual Analytics) 是为了更简便的可视分析而生的技术框架。 其名称中的第一个 **A** 具有多重涵义：一个智能驱动（*AI driven*）、自动化（*Automated*）、支持增强分析（*Augmented*）的可视分析解决方案。
-
+## ❓ 什么是AVA
+[AVA](https://github.com/antvis/AVA)（<img src="https://mdn.alipayobjects.com/huamei_qa8qxu/afts/img/A*QzIsSrfsCW0AAAAAAAAAAAAADmJ7AQ/original" width="16"> Visual Analytics）是为了更简便的可视分析而生的技术框架。第一个 **A** 具有多重涵义：智能驱动（*AI driven*）、自动化（*Automated*）、支持增强分析（*Augmented*）, **VA** 代表可视分析。它可以辅助用户进行**数据处理**、提取**数据洞察**、实现**图表的推荐和自动生成**，此外它还可以**优化已有的图表**。
 <br />
 
 <div align="center">
-  <img width="800" src="https://mdn.alipayobjects.com/huamei_qa8qxu/afts/img/A*ZINwQ6ubADQAAAAAAAAAAAAADmJ7AQ/original" alt="AVA pipeline">
+  <img width="800" src="https://mdn.alipayobjects.com/huamei_kjfwsg/afts/img/A*wpo3RpdULnAAAAAAAAAAAAAADvmcAQ/original" alt="AVA pipeline">
 </div>
 
-## npm 包
-
-### [@antv/ava](https://github.com/antvis/AVA/blob/master/packages/ava)
-
-### [@antv/ava-react](https://github.com/antvis/AVA/tree/master/packages/ava-react)
-
-## 参与贡献 [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
+## 📁 AVA的组成
+AVA包含2个包，分别为 [@antv/ava](https://www.npmjs.com/package/@antv/ava) 和 [@antv/ava-react](https://www.npmjs.com/package/@antv/ava-react)：
++ [@antv/ava](https://www.npmjs.com/package/@antv/ava) ：AVA核心包，包含四大主要模块：
```

Comment:
```suggestion
* [@antv/ava](https://www.npmjs.com/package/@antv/ava) ：AVA 核心包，包含四大主要模块：
```

check the `.markdownlint.json` of the repo, we only allow asterisk for markdown ul-style.

btw, you may want to install the markdownlint extension in vscode to show you the linting.

Reply:
done～